.github/workflows/canister-tests.yml

```diff
@@ -491,35 +491,14 @@ jobs:
           # Install npm deps
           npm ci
 
-          # Create a fake curl so that we use this build's wasm and did files; also create "witness"
-          # files to make sure the fake curl was run (rm $witness_foo will fail otherwise)
-          witnesses=$(mktemp -d); witness_wasm="$witnesses/wasm"; witness_did="$witnesses/did"
-          curl_dir=$(mktemp -d)
-
-          cat > "$curl_dir/curl" << EOF
-          #!/usr/bin/env bash
-          case "\$*" in
-            *"internet_identity.wasm"*)
-              cp "$GITHUB_WORKSPACE"/internet_identity_dev.wasm ./internet_identity.wasm && touch $witness_wasm
-              ;;
-            *"internet_identity.did"*)
-              cp "$GITHUB_WORKSPACE"/src/internet_identity/internet_identity.did ./internet_identity.did && touch $witness_did
-              ;;
-            *)
-              echo "unexpected arguments: \$*"
-              exit 1
-              ;;
-          esac
-          EOF
-
-          chmod +x $curl_dir/curl
-          PATH=$curl_dir:$PATH dfx deploy --no-wallet --argument '(null)'
-          rm -rf "$curl_dir"; rm "$witness_wasm"; rm "$witness_did"; rmdir "$witnesses"
+          ii_wasm="$GITHUB_WORKSPACE"/internet_identity_dev.wasm
```

Comment:
the $GITHUB_WORKSPACE is a bit confusing. This is just `$PWD` right?

also, shouldn't this be `file://...`? or does dfx treat anything non `https://...` as a local file? here we want to make sure that files can be pulled from a URL, should we then use a URL (i.e. `file://`)?

Reply:
Yes it is before we change directory.

`file://` is not supported. But I think replacing this here with a local file reference is fine. If the setup using `http://` does not work, this would just be a bug in `dfx`. I would assume that the SDK team has their own tests for that.